x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -40,21 +40,21 @@ export const AlertIndex = memo(() => {
       const row = json[rowIndex];
 
       if (columnId === 'alert_type') {
-        return row.value.source.endgame.metadata.key;
+        return 'Malicious File';
```

Comment:
Could you replace w/ something like:


```js
import { i18n } from '@kbn/i18n';
// ...

// The id is basically just made up. It'll be extracted and shown to the translator though
i18n.translate('xpack.endpoint.application.endpoint.alerts.alertType.maliciousFileDescription', {
  defaultMessage: 'Malicious File',
});
```
You can read more about it here:
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md
and
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md

Reply:
@oatkiller We should document these translation paths somewhere... I like the structure you've suggested here, as it's flexible enough to allow for multiple "Endpoint Team" apps in the future...